js/util/util.js

```diff
@@ -41,6 +41,14 @@ exports.wrap = function (n, min, max) {
     return n === max ? n : ((n - min) % d + d) % d + min;
 };
 
+exports.coalesce = function() {
```

Comment:
`coalesce` is a [pretty](http://msdn.microsoft.com/en-us/library/ms190349.aspx) [standard](http://www.postgresql.org/docs/8.1/static/functions-conditional.html) [name](http://dev.mysql.com/doc/refman/5.0/en/comparison-operators.html#function_coalesce) for this function.